_posts/2019-04-26-compatibility-table.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title:  "Ruby & Rails Compatibility Table"
+date: 2019-04-26 13:00:00
+categories: ["ruby", "rails", "versions"]
+author: "etagwerker"
+---
+
+This is a short post to show the compatibility between [Ruby on Rails](https://rubyonrails.org)
+and [Ruby](https://www.ruby-lang.org/en/) across different versions. In the
+process of upgrading really old applications to more modern versions of Ruby and
+Rails we have ran into a lot of these combinations.
+
+<!--more-->
+
+<table id="ruby-rails-compatibility">
+  <thead>
+    <tr>
+      <td><a href="https://rubygems.org/gems/rails/versions" style="color: white"> Rails<br/> Version </a></td>
+      <td><a href="https://www.ruby-lang.org/en/downloads/releases/" style="color: white"> Required<br/>Ruby<br/>Version</a></td>
+      <td>Recommended<br/>Ruby<br/>Version</td>
+      <td><a href="https://rubygems.org/gems/rubygems-update/versions" style="color: white"> Required<br/>Rubygems<br/>Version </a></td>
+      <td><a href="https://guides.rubyonrails.org/maintenance_policy.html" style="color: white"> Status </a></td>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>6.0.0</td>
+      <td>&gt;= 2.5.0</td>
+      <td></td>
+      <td>&gt;= 1.8.11</td>
+      <td>RC</td>
+    </tr>
+    <tr>
+      <td>5.0.0 to 5.2.x</td>
+      <td>&gt;= 2.2.2</td>
+      <td></td>
+      <td>&gt;= 1.8.11</td>
+      <td>Maintained</td>
+    </tr>
+    <tr>
+      <td>4.2.x</td>
+      <td>&gt;= 1.9.3</td>
+      <td>2.2</td>
+      <td>&gt;= 1.8.11</td>
+      <td>Maintained</td>
+    </tr>
+    <tr class="eol">
+      <td>4.1.x to 4.2.0</td>
+      <td>&gt;= 1.9.3</td>
+      <td>2.1</td>
+      <td>&gt;= 1.8.11</td>
+      <td>EOL</td>
+    </tr>
+    <tr class="eol">
+      <td>4.0.5 to 4.1.0.rc2</td>
+      <td>&gt;= 1.9.3</td>
+      <td></td>
+      <td>&gt;= 1.8.11</td>
+      <td>EOL</td>
```

Comment:
@ercohen14 Source?

Reply:
I believe the source for this one would be this page: https://weblog.rubyonrails.org/2016/6/30/Rails-5-0-final/

> Note: As per our maintenance policy, the release of Rails 5.0 will mean that bug fixes will only apply to 5.0.x, regular security issues to 5.0.x and 4.2.x, and severe security issues also to 5.0.x and 4.2.x (but when 5.1 drops, to 5.1.x, 5.0.x, and 4.2.x). This means 4.1.x and below will essentially be unsupported! Ruby 2.2.2+ is now also the only supported version of Rails 5.0+.